lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.html

```diff
@@ -3,6 +3,12 @@
         <adf-search-widget-chip [category]="category"></adf-search-widget-chip>
     </ng-container>
 
+    <ng-container *ngIf="facetFiltersService.tabbedFacet && showContextFacets">
+        <adf-search-facet-chip-tabbed
+            [tabbedFacet]="facetFiltersService.tabbedFacet"
+            [attr.data-automation-id]="'search-fact-chip-tabbed' + facetFiltersService.tabbedFacet.fields.concat('')" >
```

Comment:
Maybe we should add hypens (-) here between 'search-facet-chip-tabbed' and the fields,  as well as between the fields themselves. That way we would have an automation id like 
` search-facet-chip-tabbed-created-modified `
instead of 
` search-facet-chip-tabbedcreatedmodified `

Having said that, adding hyphens between the fields can be tricky as we don't want to end up with a trailing hyphen like 
` search-facet-chip-tabbed-created-modified- `